shell/cc-shell-category-view.c

```diff
@@ -138,7 +138,6 @@ cc_shell_category_view_constructed (GObject *object)
   gtk_icon_view_set_text_column (GTK_ICON_VIEW (iconview), COL_NAME);
   gtk_icon_view_set_item_width (GTK_ICON_VIEW (iconview), 100);
   cc_shell_item_view_update_cells (CC_SHELL_ITEM_VIEW (iconview));
-  gtk_icon_view_set_columns (GTK_ICON_VIEW (iconview), 6);
```

Comment:
I don't think so. Tested with multiple resolutions in a VM, and it seems to work as usual.